custom/echis_reports/ucr/reports/6_2_Child_Health__CH_EPI_.json

```diff
@@ -0,0 +1,550 @@
+{
+	"domains": [
+		"fmoh-echis",
+		"fmoh-echis-staging",
+		"fmoh-echis-link-test"
+	],
+	"server_environment": [
+		"production",
```

Comment:
why do you need this to run on production?

Reply:
we wanted to create a new report on the linked app. The linked app reports needs the same static reports on commcarehq to work. 
And Also all the previous reports on https://github.com/dimagi/commcare-hq/tree/master/custom/echis_reports/ucr/reports have that.

Reply:
OK, thanks